Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -315,7 +315,10 @@ void Jit64::dcbz(UGeckoInstruction inst)
 	MOV(32, R(RSCRATCH), gpr.R(b));
 	if (a)
 		ADD(32, R(RSCRATCH), gpr.R(a));
-	AND(32, R(RSCRATCH), Imm32(~31));
+	if (SConfig::GetInstance().m_LocalCoreStartupParameter.bWii)
+		AND(32, R(RSCRATCH), Imm32(~31));
+	else
+		AND(32, R(RSCRATCH), Imm32(Memory::RAM_MASK & ~31));
```

Comment:
There's no way blindly masking the address passed to dcbz like this is right: this is an untranslated address, and even it were a physical address, there are legitimate targets for dcbz outside of primary RAM, like the EFB or locked L1.

Perhaps the issue is an unusual BAT configuration?